cmds/core/netstat/netstat_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestEvalFlags(t *testing.T) {
+	for _, tt := range []struct {
+		name   string
+		ops    string
+		af     string
+		expErr error
+	}{
+		{
+			name:   "SucessSocketsDefault",
+			ops:    "sockets",
+			expErr: nil,
+		},
+		{
+			name:   "SucessSocketsIPv4",
+			ops:    "sockets",
+			af:     "ipv4",
+			expErr: nil,
+		},
+		{
+			name:   "SucessSocketsIPv6",
+			ops:    "sockets",
+			af:     "ipv6",
+			expErr: nil,
+		},
+		{
+			name:   "SucessRoute",
+			ops:    "route",
+			expErr: nil,
+		},
+		{
+			name:   "SuccessInterfaces",
+			ops:    "interfaces",
+			expErr: nil,
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			switch tt.ops {
+			case "route":
+				*routeFlag = true
+			case "interfaces":
+				*interfacesFlag = true
+			case "stats":
+				*statsFlag = true
+			case "sockets":
+				switch tt.af {
+				case "ipv4":
+					*ipv4Flag = true
+				case "ipv6":
+					*ipv6Flag = true
+				}
+			case "groups":
+				*groupsFlag = true
+			}
+			if err := evalFlags(); !errors.Is(err, tt.expErr) {
+				t.Errorf("evalFlags() failed: %v, want: %v", err, tt.expErr)
+			}
+
+			resetFlags()
+		})
+	}
+}
+
+func resetFlags() {
+	// Info source flags
+	*routeFlag = false
```

Comment:
if you adapt the idea of using a struct, you won't need resetFlags.

Reply:
true. Already working on the struct.